### PR TITLE
update Excel tutorial to use onReady

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
The repo that contains the starting state of the tutorial is also be updated to use onReady.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-tutorial-open-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/onready-in-excel-tutorial/docs/includes/excel-tutorial-open-dialog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811171916351534-566%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OoibusYcwRVdvVZHy68bEJRxjQb5fsL%2Fr0PrgJi93DQ%3D&st=2018-11-17T19%3A12%3A50Z&se=2018-12-18T19%3A17%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/excel-tutorial-open-dialog.md [hunk 1]
```diff
@@ -45,20 +45,22 @@ In this final step of the tutorial, you'll open a dialog in your add-in, pass a
...
     "use strict";
 
-        Office.initialize = function() {
-            $(document).ready(function () {  
+        Office.onReady()
+            .then(function() {
+                $(document).ready(function () {  
 
-                // TODO1: Assign handler to the OK button.
+                    // TODO1: Assign handler to the OK button.;
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/excel-tutorial-open-dialog.md]: why put a ; here?

File: docs/includes/excel-tutorial-open-dialog.md [hunk 2]
```diff
@@ -45,20 +45,22 @@ In this final step of the tutorial, you'll open a dialog in your add-in, pass a
...
-   - The jQuery `ready` function is called inside the `initialize` method. It is an almost universal rule that the loading, initializing, or bootstrapping code of other JavaScript libraries should be inside the `Office.initialize` function.
+
+   - *Every page that calls APIs in the Office.JS library must first ensure that the library is fully initialized.* The best way to do that is to call the `Office.onReady()` method. If your add-in has its own initialization tasks, the code should go in a `then()` method that is chained to the call of `Office.onReady()`. For an example, see the app.js file in the project root. The call of `Office.onReady()` must run before any calls to Office.JS; hence the assignment is in a script file that is loaded by the page, as it is in this case.
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/excel-tutorial-open-dialog.md]: Should these bullets be in a [!NOTE]?

Comment (Rick-Kirkham) [reply docs/includes/excel-tutorial-open-dialog.md]: These should be bullets. They explain the "why" of things in the code sample that follows.

File: docs/includes/excel-tutorial-open-dialog.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/includes/excel-tutorial-open-dialog.md]: typo. fixed

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-tutorial-open-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/onready-in-excel-tutorial/docs/includes/excel-tutorial-open-dialog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811192035478267-566%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=V9ZvwQf2eL0zKUjQ%2FUQ5%2FKedA5p9AqrJXgIaVV00dpU%3D&st=2018-11-19T20%3A32%3A11Z&se=2018-12-20T20%3A37%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/excel-tutorial-open-dialog.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/includes/excel-tutorial-open-dialog.md]: What I meant was put the bullets in a note. The line above it ends with "Note:" hence why I asked.

Comment (Rick-Kirkham) [reply docs/includes/excel-tutorial-open-dialog.md]: I see. The "Note:" was meant to be a more concise version of "Note the following in this code:". I'll clarify.

**Comment by Rick-Kirkham** [comment]
Reverting because there's a bug in WAC that requires Office.initialize, so the tutorial won't work if the reader sideloads on WAC.

**Pull request merged**